src/components/Slider/helpers.js

```diff
@@ -8,7 +9,7 @@ export function createSlidesXPoints(
   //prevent console errors
   if (!sliderContainer || !sliderContent) return null;
 
-  return new Array(slidesCount).fill().map((_, i) => {
+  return map(Array(slidesCount), (_, i) => {
```

Comment:
well, map it's not the best cause to use lodash
`Array(slidesCount).map((_, i) => {});` will do the same

Reply:
Not really, we need to fill an array if we use default map

Reply:
Like before

Reply:
But it's really not the best cause, so I'm gonna change it back...

Reply:
I have two variants for you 😏 
`Array.from({length:7},(_, i) => i)`
`[...Array(7)].map((_, i) => i)`
First one looks longer, but I like it more
You can go with your variant, or consider mines )